src/org/labkey/test/tests/component/GridPanelTest.java

```diff
@@ -1612,18 +1616,21 @@ public void testExport() throws IOException, CommandException
 
             List<Map<String, Object>> expectedData = new ArrayList<>();
 
-            List<String> intColData = grid.getColumnDataAsText(FILTER_INT_COL);
+            List<String> intColData = grid.getColumnDataAsText(FILTER_INT_COL.getLabel());
```

Comment:
This supports `FieldInfo`
```suggestion
            List<String> intColData = grid.getColumnDataAsText(FILTER_INT_COL);
```

Reply:
Done